lib/json5.js

```diff
@@ -124,9 +125,9 @@ JSON5.parse = (function () {
             return key;
         },
 
-        number = function () {
+        _number = function () {
```

Comment:
I'd like to use a better name than `_number` here. Let's come up with a names that describe the difference between `number()` and `_number()`.